docs/conf/atlantic/2024/news/welcome.rst

```diff
@@ -0,0 +1,68 @@
+:template: {{year}}/generic.html
+
+.. post:: January 4, 2024
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+Welcome to the official launch of `Write the Docs {{ city }} {{year}} <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/>`_.
+We're excited to announce the dates for our next conference: **Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**.
+
+Following the success of last year's Atlantic conference, we will be running a **virtual event** again,
+with a similar schedule.
+
+The schedule will run from the
+early afternoon to early evening in most European/African timezones,
+and from early morning to late afternoon in US East Coast/South America timezones.
+Of course, this being a virtual conference, attendees and speakers
+are welcomed from anywhere in the world.
+
+Sign up to the separate
+`Atlantic conference mailing list <https://www.writethedocs.org/newsletter/>`_ to stay
+up to date on the call for proposals, ticket sales and all other updates.
+
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+**We plan to launch our CFP in early May 2024** for those who are excited to speak.
```

Comment:
What do we think about opening the Atlantic CFP during Portland in April? Could be a good opportunity to announce and promote and schmooze about it earlier

Reply:
yeah, I think this makes sense. Straightforward enough to set up, and because it's virtual, folks can more easily submit from Portland to Atlantic